res_users_case_insensitive/models/res_users.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2015-TODAY LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api, fields
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+    login = fields.Char(
+        'Login',
+        size=64,
```

Comment:
I don't see what's the benefit of limiting the size of the login to 64 characters. In odoo 11, this field is not limited. See this definition in the `odoo/addons/base/res/res_users.py` file :

```python
    login = fields.Char(required=True, help="Used to log into the system")
```